pkg/aspectsupport/logger/manager.go

```diff
@@ -68,30 +90,100 @@ func (m *manager) NewAspect(c *aspectsupport.CombinedConfig, a aspect.Adapter, e
 		inputs = c.Aspect.Inputs
```

Comment:
This is the stuff that is generated from an expression.  It is filled in by the operator -- to generate a value from some combination over the original source attributes.

For another example, see: https://github.com/istio/mixer/blob/master/pkg/aspectsupport/listChecker/manager.go#L109